xarizmi/candlestick.py

```diff
@@ -9,6 +9,29 @@
 
 
 class Candlestick(BaseModel):
+    """[Summary?]
+
+    :param [ParamName]: [ParamDescription], defaults to [DefaultParamVal]
```

Comment:
Shouldn't this lines be removed? Are they description of documentation style?

Reply:
Yes, they are documentation style. I keep the summary line as a placeholder to replace with a short description of the class.